### ISSUE TITLE
how can i get the enclosure content?

### ISSUE THREAD

**Issue by SkyD666** [prof18/RSS-Parser#112]
**Is your feature request related to a problem? Please describe.**
the type of the enclosure is "application/x-bittorrent", i want to get the content.

> \<enclosure url="xxx.torrent" length="0" type="application/x-bittorrent"/\>

**Describe the solution you'd like**
Please provide a more general parsing solution for enclosure

**Comment by prof18** [comment]
Hi!

I will think about adding this custom field.

**Comment by berberman** [comment]
FYI I've just made a very ad-hoc change for my own use :)
https://github.com/berberman/RSS-Parser/commit/846e24c2bca0f3333c49dface4a585c51ca8749f

**Comment by prof18** [comment]
Amazing! I'll take a look at it, and see if it makes sense to upstream it!

**Comment by prof18** [comment]
@berberman Would you mind opening a PR with your changes? I want to add and expand the feature and would love to have your contribution!

**Comment by berberman** [comment]
Yes, though I'm not sure if my changes will break the paradigm/design of the current code base.

**Comment by prof18** [comment]
no worries, I'll adapt it if necessary
